actions/bulk_link_issue.py

```diff
@@ -6,16 +8,40 @@
 
 
 class BulkLinkJiraIssueAction(BaseJiraAction):
+    def link_issues(
+        self,
+        semaphore,
+        issue_key=None,
+        target_issue=None,
+        direction=None,
+        link_type=None,
+    ):
+        with semaphore:
+            if direction == "outward":
+                outward_issue_key = issue_key
+                inward_issue_key = target_issue
+                issue = self._client.create_issue_link(
+                    link_type, inward_issue_key, outward_issue_key
+                )
 
-    def run(self, issue_key_list=None, target_issue=None, direction=None, link_type=None):
-        if direction == 'outward':
-            inward_issue_key = target_issue
-            for outward_issue_key in issue_key_list:
-                issue = self._client.create_issue_link(link_type, inward_issue_key,
-                                                       outward_issue_key)
-        if direction == 'inward':
-            outward_issue_key = target_issue
-            for inward_issue_key in issue_key_list:
-                issue = self._client.create_issue_link(link_type, inward_issue_key,
-                                                       outward_issue_key)
-        return issue
+            if direction == "inward":
+                inward_issue_key = issue_key
+                outward_issue_key = target_issue
+                issue = self._client.create_issue_link(
+                    link_type, inward_issue_key, outward_issue_key
+                )
+        print(issue)
```

Comment:
I see we are using `print()` here but we now have no `return` from `run()` with your changes. I'm doubting that anyone is parsing the return from this action given it only ever returned a status code, so that might be ok.  Even without adding a `return` I wonder if we should at least print the issue ID and its status so if a link for one issue fails we can easily debug which issue did not successfully link. So we can do something like:

```
response = {"issue": target_issue, "response": issue} 
print(response)
```

I feel that might be a nice improvement for debugging failures. What do you think?

Reply:
Thanks @floatingstatic , Apologies for delay, i was bit caught up.
 I accidentaly closed this PR while syncing the fork but made changed in new PR (https://github.com/StackStorm-Exchange/stackstorm-jira/pull/66), Kindly review, I have made updated as per your feedback, let me know. Once again thank you so much!